app/components/file-details/vulnerability-analysis-details/index.hbs

```diff
@@ -109,8 +116,13 @@
 
     {{else if (or @analysis.isRisky @analysis.isOverriddenAsPassed)}}
       {{#if (and (eq @analysis.cvssVersion 3) @analysis.cvssVector)}}
-        <div local-class='analysis-section'>
-          <AkStack data-test-analysisDetails-regulatory='cvssv3'>
+        <div
+          local-class='analysis-section {{if
+            @analysis.isOverriddenAsPassed
+            "analysis-overridded-passed"
```

Comment:
can we just have `analysis-overridded-passed` at main container level and other override child classes

Eg:
```scss
.analysis-overridded-passed {
  .analysis-section {
  // overridden styles
  }
}
```